app/views/subscriptions/_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= simple_form_for subscription do |f| %>
   <div class="form-container__body">
-    <p>But there's good news! If you'd like to provide us with your email, we will notify you once we begin service in your area:</p>
-    <%= f.input :email, label: false %>
-    <%= f.input :zipcode, as: :hidden %>
+    <p><%= t(".prompt") %></p>
+    <%= f.input :email %>
+    <%= f.input :zipcode %>
```

Comment:
@corwinharrell we should keep the `<label>` around for the sake of accessibility.

If we want to minimize the prominence of the label, let's style it with something like https://github.com/thoughtbot/sky-labels-rails.

For now, it's best to keep the zipcode input visible, since it's currently being populated by the URL, which could change. 

If it's totally necessary to hide the input, let me know and I'll cut a card.